app/controllers/user_controller.rb

```diff
@@ -17,6 +17,12 @@ def update
     end
   end
 
+  # rubocop:disable Naming/MemoizedInstanceVariableName
+  def check_email
+    @user ||= User.find(params[:id])
+  end
```

Comment:
`check_email` is an action. The assignment of `@user` is something that happens within that action. It should only be called once, so I don't think this should be memoized. I'd suggest:
```ruby
def check_email
  @user = User.find(params[:id]
end
```
And with you the rubocop disable is not longer required.

Reply:
It'll raise another similar complaint about the variable no matching the method I think

Reply:
This is why you get paid the big bucks - should have been obvious....

Reply:
Are we able to get this to follow the RESTful convention, and in so doing side step the issue?